### ISSUE TITLE
Wrong output in ONNX NerModel

### ISSUE THREAD

**Issue by gaganmanku96** [ThilinaRajapakse/simpletransformers#812]
Hi, I think there is some issue with ONNX outputs for NerModel.
I'm using 0.48.15

Regards

**To Reproduce**
![image](https://user-images.githubusercontent.com/20324385/98761600-d1e2d780-23fb-11eb-935b-378813b1020e.png)

**Comment by ThilinaRajapakse** [comment]
ONNX support is currently experimental, so it's possible. Are the predictions from the ONNX model bad for all/most examples?

**Comment by gaganmanku96** [comment]
Yes, I observed it all examples.

I actually found a solution but it is only working for few exampes. I'm not sure but I think the problem is with out_label_ids. I'm investigating more on it.
![image](https://user-images.githubusercontent.com/20324385/99762293-2bdf4d80-2b1e-11eb-9c60-e118cf4ecbc8.png)

**Comment by gaganmanku96** [comment]
@ThilinaRajapakse Yes, I observed it with all examples.

I actually found a solution for it. 
![image](https://user-images.githubusercontent.com/20324385/99763083-f471a080-2b1f-11eb-92d9-85e14f3e3200.png)

![image](https://user-images.githubusercontent.com/20324385/99763296-824d8b80-2b20-11eb-9623-2208223b24b1.png)


I compared the out_label_ids of ONNX Model and without ONNX Model and made some changes.
It is working fine for all examples tried. I'll try to train a model and if it works I'll raise a PR.
